### PR TITLE
[ADP-1963] publish mint and burn data in ApiTransaction from stored cbor

### PR DESCRIPTION
**Pull request by paolino**
I have added mint and burn fields to get /transaction and get /transactions by parsing the CBOR from the transaction history

### Issue Number

ADP-1963

### CODE REVIEW

**Comment by paolino** [comment]
bors try

File: test/e2e/spec/e2e_spec.rb [hunk 1]
```diff
@@ -944,9 +950,22 @@ def fingerprint
...
+        # Mint
+        expect(tx_decoded['mint']['tokens']).to include(have_key('assets')).exactly(3).times
+        expect(tx_decoded['mint']['tokens'].to_s).to include(policy_vkh).exactly(3).times
+        expect(tx_decoded['mint']['tokens'].to_s).to include('1000').exactly(3).times
+        expect(tx_decoded['mint']['tokens'].to_s).to include(policy_id1).once
+        expect(tx_decoded['mint']['tokens'].to_s).to include(policy_id2).once
+        expect(tx_decoded['mint']['tokens'].to_s).to include(policy_id3).once
+        expect(tx_decoded['mint']['tokens'].to_s).to include(asset_name('Token1')).once
+        expect(tx_decoded['mint']['tokens'].to_s).to include(asset_name('Token2')).once
+
```
Comment (HeinrichApfelmus) [inline test/e2e/spec/e2e_spec.rb:962]: We probably want some testing like this in Haskell as well. To be discussed.

**Review by HeinrichApfelmus — APPROVED** [review]

**Comment by HeinrichApfelmus** [comment]
bors merge

**Comment by HeinrichApfelmus** [comment]
bors retry

**Pull request closed without merging**